.all-contributorsrc

```diff
@@ -150,6 +150,16 @@
         "code",
         "test"
       ]
+    },
+    {
+      "login": "kkemple",
+      "name": "Kurtis Kemple",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/3629876?v=3",
+      "profile": "https://twitter.com/kurtiskemple",
+      "contributions": [
+        "code",
+        "doc"
```

Comment:
Want to add test here too? See the contributing guidelines for how to regenerate after you update this 😉 if your want to.